components/camera/transformpipeline/composed.go

```diff
@@ -23,6 +23,18 @@ type depthToPretty struct {
 	cameraModel    *transform.PinholeCameraModel
 }
 
+func propsFromVideoSource(ctx context.Context, source gostream.VideoSource) (camera.Properties, error) {
```

Comment:
Another helper function created for a pattern that was common in `transformationpipeline`